srml/balances/src/lib.rs

```diff
@@ -165,7 +165,12 @@ decl_storage! {
 						// <= begin it should be >= balance
 						// >= begin+length it should be <= 0
 
-						let per_block = balance / length;
+						let per_block = if length.is_zero() {
```

Comment:
could be shortened to a simple:
```suggestion
						let per_block = balance / length.max(1);
```